style.css

```diff
@@ -1 +1,156 @@
-/*  Add your styling here */
+/*  DEFAULT */
+:root {
+  --main-color: #faf1da;
+  --font-color: #5f4501;
+  --third-color: #9c6644;
+}
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: black;
```

Comment:
Could you also make this a CSS variable? You already have some, why not make it consistent? That way, in the future if you ever want to change your black font colour (for example to a lighter shade of black, say), you can just update the CSS variable and it will change everywhere.